Ingress/images/haproxy/haproxy.cfg

```diff
@@ -0,0 +1,36 @@
+# Default haproxy config file.
+global
+    daemon
+    stats socket /tmp/haproxy
+    server-state-file global
+    server-state-base /var/state/haproxy/
+
+defaults
+    log global
+    mode  http
+    option  httplog
```

Comment:
can we replace this with dontlog-normal? so I don't get 
```
[WARNING] 313/183747 (1) : config : log format ignored for proxy 'stats' since it has no log address.
[WARNING] 313/183747 (1) : config : log format ignored for frontend 'httpfrontend' since it has no log address.
```